src/app/components/header/header.component.html

```diff
@@ -1,10 +1,43 @@
-<div class="container-fluid border-bottom mb-10">
-  <div class="container-xxl pb-5 mt-5">
-    <img src="/assets/images/logo/freshcart-logo.svg" class="pb-5" />
-    <ul class="nav">
-      <li *ngFor="let item of categoriesList$ | async" class="nav-item">
-        <a [routerLink]="'/categories/'+ item.id" class="nav-link">{{item.name}}</a>
-      </li>
-    </ul>
+<header class=" pb-2" [ngClass]="(isMobileMenuVisible$ | async) ? '' : 'border-bottom'">
```

Comment:
You can do this in this way: [ngClass]="{ class-name: Condition }
If Condition is true, class-name exist
if Condition is false, class-name not exist